src/services/formatting/ruleOperation.ts

```diff
@@ -1,4 +1,5 @@
 ///<reference path='references.ts' />
+/* tslint:disable:no-null-keyword */

```

Comment:
Weird. Is this the only file that uses null? I wonder if it was intentional. Does it work if you switch to undefined? Ron or Vlad might also be able to tell you if the usage was intentional.